### PR TITLE
Add tasks descriptions

### PR DESCRIPTION
**Pull request by nebilbenmabrouk**

### CODE REVIEW

File: BigData/resources/catalog/Azure_HDInsight_Spark_Send_Signal_${SIGNAL_TO_SEND}.xml [hunk 1]
```diff
@@ -24,6 +24,9 @@
   </genericInformation>
   <taskFlow>
     <task name="find_job" fork="true">
+      <description>
+        <![CDATA[ A task that finds a running job (managing a HDInsight Spark cluster) based on the associated workflow name and the value of a key variable. ]]>
```
Comment (mklkun) [inline BigData/resources/catalog/Azure_HDInsight_Spark_Send_Signal_${SIGNAL_TO_SEND}.xml:28]: a HDInsight => an HDInsight

**Review by mklkun — APPROVED** [review]
A small typo.

**Review by Hiba-Alili — APPROVED** [review]

File: BigData/resources/catalog/Azure_HDInsight_Spark_Send_Signal_${SIGNAL_TO_SEND}.xml (hunk 1 above)
Comment (nebilbenmabrouk) [reply BigData/resources/catalog/Azure_HDInsight_Spark_Send_Signal_${SIGNAL_TO_SEND}.xml:28]: Fixed, thanks

**Comment by nebilbenmabrouk** [comment]
re-test this please

**Pull request merged**